Jenkinsfile

```diff
@@ -104,16 +113,17 @@ spec:
 
                     // the network=host needed to download dependencies using the host network (since we are inside 'docker'
                     // container)
-                    sh "docker build --network=host -f Dockerfile . -t icgcargo/program-service:latest"
+                    sh "docker build --network=host -f Dockerfile . -t icgcargo/program-service:latest -t icgcargo/program-service:${version}"
+                    sh "docker push icgcargo/program-service:${version}"
                     sh "docker push icgcargo/program-service:latest"
                     withCredentials([usernamePassword(credentialsId: 'argoGithub', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USERNAME')]) {
-                        sh "git tag ${commit}"
+                        sh "git tag ${version}"
                         sh "git push https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/icgc-argo/program-service --tags"
                     }
                 }
                 build(job: "/ARGO/provision/program-service", parameters: [
                      [$class: 'StringParameterValue', name: 'AP_ARGO_ENV', value: 'qa' ],
-                     [$class: 'StringParameterValue', name: 'AP_ARGS_LINE', value: "--set-string image.tag=${commit}" ]
+                     [$class: 'StringParameterValue', name: 'AP_ARGS_LINE', value: "--set-string image.tag=${version}" ]
```

Comment:
guess we could remove this build parameter, so that the job uses what ever version is in the helm values??